packages/category-exporter/package.json

```diff
@@ -1,18 +1,15 @@
 {
-  "private": false,
   "publishConfig": {
     "access": "public"
   },
   "engines": {
-    "node": ">=14"
+    "node": ">=18"
   },
   "name": "@commercetools/category-exporter",
   "version": "4.0.0",
   "description": "Exports categories from the commercetools platform.",
   "main": "lib/main.js",
-  "bin": {
-    "category-exporter": "bin/category-exporter.js"
-  },
+  "bin": "bin/category-exporter.js",
```

Comment:
Can you explain this change? 🙏

Reply:
Not sure I did this. Must have been some tooling. Let me revert.

Reply:
[471ba06](https://github.com/commercetools/nodejs/pull/1930/commits/471ba060beef74d56f125daa3615be177a582d7d)